gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfigurationTest.java

```diff
@@ -29,6 +33,7 @@
 import static com.google.cloud.hadoop.util.HadoopCredentialConfiguration.USER_IMPERSONATION_SERVICE_ACCOUNT_SUFFIX;
 import static com.google.cloud.hadoop.util.testing.HadoopConfigurationUtils.getDefaultProperties;
 import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertEquals;
```

Comment:
We use Truth asserts instead of JUnit asserts in all our tests, please use them for new tests too.

Reply:
Done